libs/src/userStateManager.ts

```diff
@@ -1,14 +1,25 @@
-const CURRENT_USER_EXISTS_LOCAL_STORAGE_KEY = require('./constants').CURRENT_USER_EXISTS_LOCAL_STORAGE_KEY
+import { CURRENT_USER_EXISTS_LOCAL_STORAGE_KEY } from './constants'
 
-const supportsLocalStorage = () => typeof window !== 'undefined' && window && window.localStorage
+const supportsLocalStorage = () =>
+  typeof window !== 'undefined' && window && window.localStorage
+
+export type CurrentUser = {
```

Comment:
some day this should conform to some other type we have for User

Reply:
yeah good call, adding TODO